.gitignore

```diff
@@ -2,6 +2,7 @@
 
 jsonnet/vendor/
 tmp/
+.idea/
```

Comment:
Consider using global `.gitignore` for IDE-specific files. More in https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Removed